src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.hpp

```diff
@@ -0,0 +1,138 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace Elasticity::ConstitutiveRelations {
+
+/*!
+ * \brief A cubic crystalline material
+ *
+ * \details For a cubic crystalline material the Elasticity tensor in the linear
+ * constitutive relation \f$T^{ij}=-Y^{ijkl}S_{kl}\f$ reduces to
+ *
+ * \f[
+ * Y^{ijkl} = \frac{1 - \nu}{\nu}\lambda \delta^{ij}\delta^{kl}\delta^{jk} +
+ * \lambda \delta^{ij}\delta^{kl} (1 - \delta^{jk}) + \mu
+ * (\delta^{ik}\delta^{jl} + \delta^{il}\delta^{jk})(1 - \delta^{ij})
+ * \f]
+ *
+ * with the three independent parameters: the _Lamé parameter_ \f$\lambda\f$,
+ * the _Shear modulus_ \f$\mu\f$ and the _Poisson ratio_ \f$\nu\f$. In the
```

Comment:
you need to use `%Poisson` to avoid a link to the Poisson equation documentation